multi-data-centers-in-one-city-deployment.md

```diff
@@ -71,6 +71,9 @@ member leader_priority pdName2 4
 member leader_priority pdName3 3
 ```
 
+> **注意**
+> 5.2 以上版本已不支持，请使用 [Placement Rules](/configure-placement-rules.md) 替代该功能。

```

Comment:
I think we still support it but the label-scheduler is not added by default.

Reply:
```suggestion
> TiDB 5.2 及以上版本不支持通过以上模板设置调度策略。若要设置调度策略，请使用 [Placement Rules](/configure-placement-rules.md)。

```